libs/azure-api-client/composer.json

```diff
@@ -51,6 +51,7 @@
             "@infection"
         ],
         "phpcs": "phpcs -n --ignore=vendor,cache,Kernel.php --extensions=php .",
+        "phpcbf": "phpcbf --extensions=php src tests",
```

Comment:
Nemelo by to mit stejnu konfiguraci jako `phpcs`? Cili `-ignore=vendor,cache,Kernel.php --extensions=php .`?

Reply:
Nekde to mame takhle i pro phpcs, jsem to zkopiroval z jiny libky :)
Asi by to mohlo byt stejny. Ale vlastne zrovna ignorovat cache a Kernel je blbost, kdyz libky nic takoveho nemaji :-)

Reply:
Pravda.. takze mozna naopak upravit ten `phpcs` podle `phpcbf` :smile: Kazdopadne ale citim shodu na tom, ze by ty dva commandy mely mit stejnou konfiguraci.

Reply:
Jo ja to poslu v dalsim PR